autoapi/templates/python/function.rst

```diff
@@ -1,6 +1,6 @@
 {%- if obj.display %}
 
-.. function:: {{ obj.name }}({{ obj.args|join(',') }})
+.. function:: {{ obj.short_name }}({{ obj.args|join(',') }})
```

Comment:
We should ensure linking is correct with this change. We are not currently testing whether this is nested correctly.